### PR TITLE
fix(tornado.py): fix response parsing

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: epsagon/runners/tornado.py [hunk 1]
```diff
@@ -96,6 +96,10 @@ def update_response(self, response, response_body=None):
         )
 
         if response_body:
+            if isinstance(response_body, list):
+                response_body = response_body[0]
```
Comment (adavidai) [inline epsagon/runners/tornado.py:100]: Is it possible that the passed response_body is the actual response that the user uses? 
Could be a good idea to use a different var and not **change** the response_body itself

File: epsagon/runners/tornado.py [hunk 2]
```diff
@@ -96,6 +96,10 @@ def update_response(self, response, response_body=None):
...
+            if isinstance(response_body, bytes):
+                response_body = response_body.decode('utf-8')
```
Comment (adavidai) [inline epsagon/runners/tornado.py:102]: same

File: epsagon/runners/tornado.py (hunk 1 above)
Comment (adavidai) [reply epsagon/runners/tornado.py:100]: Also handle empty list use case

**Comment by ranrib** [comment]
Addressed in https://github.com/epsagon/epsagon-python/pull/317

**Pull request closed without merging**